tests/epyccel/test_epyccel_variable_annotations.py

```diff
@@ -10,6 +10,14 @@
 from pyccel.errors.errors import PyccelSemanticError, Errors
 from pyccel.decorators import allow_negative_index, stack_array
 
+def parametrize_languages():
+    return pytest.mark.parametrize('language', [
+        pytest.param("c", marks=pytest.mark.c),
+        pytest.param("fortran", marks=[
+            pytest.mark.skip(reason="Variable declaration not implemented in fortran"),
```

Comment:
Please reference the issue that will fix this (this makes it easier to find all xfails that should be removed when you fix the issue as you just need to search for the number)

Reply:
updated cd22d9552fa7c174faa97aaba068fe8ff2acd247